app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :post
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] This will give error, because in schema you have `user` as `author`, to make this assosiation work you should add  foreign key:
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

```
- [ ] For relationship has_many, other model should be in plural, `has_many :post` will give error, should be `has_many :posts`